### PR TITLE
KIALI-1132 IstioConfig: MixerRules UX refactoring

### PR DESCRIPTION
**Pull request by lucasponce**
- Present MixerRules in a list/table instead of a list of links
- Separate AceEditor from list/table view for MixerRules

istioconfig ogv

### CODE REVIEW

**Comment by abonas** [comment]
Shouldn't all the fields in the table be aligned to left instead of center?

File: src/pages/IstioConfigDetails/IstioRuleInfo.tsx [hunk 1]
```diff
@@ -162,10 +156,7 @@ class IstioRuleInfo extends React.Component<IstioRuleInfoProps> {
             formatters: [this.headerFormat]
           },
           cell: {
-            formatters: [this.cellFormat],
-            props: {
-              align: 'text-center'
-            }
+            formatters: [this.cellFormat]
```
Comment (abonas) [inline src/pages/IstioConfigDetails/IstioRuleInfo.tsx]: where does it take the alignment style from? I see you deleted the text-center, so the left alignment is taken from somewhere else?

**Comment by lucasponce** [comment]
Table columns fixed:
image

**Comment by lucasponce** [comment]
@xeviknal Travis seems happy now.

File: src/pages/IstioConfigDetails/IstioRuleInfo.tsx (hunk 1 above)
Comment (lucasponce) [reply src/pages/IstioConfigDetails/IstioRuleInfo.tsx]: It is the default alignment style from the component.
Is it looking different locally to you ?

**Pull request merged**